packages/zoe/src/zoeService/startInstance.js

```diff
@@ -63,19 +63,24 @@ export const makeStartInstance = (
 
     const instance = makeInstanceHandle();
 
-    const snoozedStrings = [];
+    /** @type {Readonly<string[]>} */
+    let snoozedStrings = [];
     // if any string in the list matches, don't process the invitation
     const isSnoozed = string =>
       snoozedStrings.some(s => string.indexOf(s) > -1);
     const updateSnoozedList = strings => {
+      assert(Array.isArray(strings), `${q(strings)} must be an Array`);
+      const proposedStrings = Array.from(strings);
       assert(
-        strings.every(s => typeof s === 'string' && isAlphanumeric.test(s)),
+        proposedStrings.every(
+          s => typeof s === 'string' && isAlphanumeric.test(s),
+        ),
         X`Snoozed strings (${q(
           strings,
-        )}) must be non-empty alpha-numeric strings.`,
+        )}) must be a (possibly empty) Array of alpha-numeric strings.`,
```

Comment:
suffices to say "must be an Array of alpha-numeric strings"

Reply:
pointing out how to reset the list is good. I'd expect to see that in the function jsdoc.  I'd probably want to know that before I send an invalid argument :)

Reply:
Restrictions on the string contents are gone. Resolving.